src/SprykerEco/Service/ComputopApi/Converter/ComputopApiConverter.php

```diff
@@ -20,12 +22,26 @@ class ComputopApiConverter implements ComputopApiConverterInterface
      */
     protected $computopApiConfig;
 
+    /**
+     * @var \SprykerEco\Service\ComputopApi\Mapper\ComputopApiMapperInterface
+     */
+    protected ComputopApiMapperInterface $computopApiMapper;
```

Comment:
```suggestion
    protected $computopApiMapper;
```

class properties typehinting is forbidden for now

Reply:
Fixed